diagnostic.md

```diff
@@ -32,14 +36,18 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+$("#firstHeader").replacewith(silly name)
+$("#h1").html(I think this is right?)
+$("#firstHeader").replacewith()
+$("#h1").html()
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
+The dom is a place in the browser where a base copy of the site you're looking at is stored and can be manipulated. However what you can access is a "copy" of the websites skeleton. It keeps the master, and gives you one to change, since you can't change a site you don't have access to. 
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.